frontend/pages/admin/AppSettingsPage/AppSettingsPage.jsx

```diff
@@ -71,7 +71,7 @@ class AppSettingsPage extends Component {
       <div className={`${baseClass} body-wrap`}>
         <h1>App Settings</h1>
         <WarningBanner
-          message="Email is not currently configured in Fleet. Many features rely on email to work."
+          message="SMTP is not currently configured in Fleet. The &quot;Add new user&quot; features requires that SMTP is configured in order to send invitation emails."
```

Comment:
```suggestion
          message="SMTP is not currently configured in Fleet. The &quot;Add new user&quot; feature requires that SMTP is configured in order to send invitation emails. Users may also be added with &quot;fleetctl user create&quot;."
```

Reply:
>Users may also be added with `fleetctl user create`

Ah I didn't know this. I wonder if the SMTP warning message is too visible even after these changes if users can always be added via `fleetctl`. Maybe the "Add new user" button is never disabled but selecting it when SMTP hasn't been configured displays the warning inside the "Add new user" modal. Something to discuss.

Committing your copy changes for now.

Reply:
That's a good idea. I agree, for now let's just go with the copy changes and we can consider how to improve the UX further.